### PR TITLE
Grpc authentication tests

### PR DESCRIPTION
**Pull request by azevaykin**
### Changelog entry 

More tests for GRPC authentification 

### Changelog category 

* Not for changelog (changelog entry is not required)

### Additional information

...

### CODE REVIEW

File: ydb/services/ydb/ydb_login_ut.cpp [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+        loginConnection.CreateUser(User, Password);
+
+        auto factory = CreateLoginCredentialsProviderFactory({.User = User, .Password = "WrongPassword"});
+        auto loginProvider = factory->CreateProvider(loginConnection.GetCoreFacility());
+        UNIT_ASSERT_EXCEPTION_CONTAINS(loginProvider->GetAuthInfo(), yexception, "Invalid password");
+
+        loginConnection.Stop();
+    }
+
+    Y_UNIT_TEST(DisableBuiltinAuthMechanism) {
```
Comment (molotkov-and) [inline ydb/services/ydb/ydb_login_ut.cpp:132]:
```suggestion
    Y_UNIT_TEST(DisableLoginAuthentication) {
```
Builtin auth mechanist it is about tokens with `@builtin` domain

File: ydb/services/ydb/ydb_login_ut.cpp [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+        auto factory = CreateLoginCredentialsProviderFactory({.User = User, .Password = Password});
+        auto loginProvider = factory->CreateProvider(loginConnection.GetCoreFacility());
+        TStringBuilder expectedErrorMessage;
+        UNIT_ASSERT_EXCEPTION_CONTAINS(loginProvider->GetAuthInfo(), yexception, "Login authentication is disabled");
+
+        loginConnection.Stop();
+    }
+
+    Y_UNIT_TEST(NoDescribeRights) {
```
Comment (molotkov-and) [inline ydb/services/ydb/ydb_login_ut.cpp:143]: In this test user has not connect rights also. Are you sure you're checking for the absence of the correct right?

File: ydb/services/ydb/ydb_login_ut.cpp (hunk 1 above)
Comment (azevaykin) [reply ydb/services/ydb/ydb_login_ut.cpp:132]: Done

File: ydb/services/ydb/ydb_login_ut.cpp (hunk 2 above)
Comment (azevaykin) [reply ydb/services/ydb/ydb_login_ut.cpp:177]: fc7e6eaa976a27155ef2f9f2af952b1c0e151a03

**Review by molotkov-and — APPROVED** [review]

**Pull request merged**